src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -93,6 +95,32 @@ object DUOS {
     }
   }
 
+
```

Comment:
I shoved this in DUOS.scala because that's what made sense to me - but I can move it elsewhere.

Reply:
I put mine in `DataUse.scala`, mostly because it was so closely related to something already in there. I'm also not sure if that's where it belongs.